com.github.coslyk.MoonPlayer.json

```diff
@@ -0,0 +1,169 @@
+{
+  "app-id": "com.github.coslyk.MoonPlayer",
+  "runtime": "org.kde.Platform",
+  "runtime-version": "5.11",
+  "sdk": "org.kde.Sdk",
+  "command": "moonplayer",
+  "finish-args": [
+      "--share=ipc",
+      "--socket=x11",
+      "--socket=wayland",
+      "--device=all",
+      "--share=network",
+      "--socket=pulseaudio",
+      "--filesystem=host",
+      "--env=LC_NUMERIC=C"
+  ],
+  "modules": [
+    {
+      "name": "moonplayer",
+      "buildsystem": "qmake",
+      "sources": [
+        {
+          "type": "archive",
+          "url": "https://github.com/coslyk/moonplayer/archive/2.1.1.tar.gz",
+          "sha256": "3c27895962f064d69d45b8c4ea159a20cbc5423db4f34080066bf23d9d3bf43a"
+        }
+      ],
+      "subdir": "src",
+      "modules": [
+        {
+          "name": "qtermwidget",
+          "cleanup": [ "/include", "/lib/pkgconfig", "/lib/cmake" ],
+          "buildsystem": "cmake-ninja",
+          "build-options": {
+		    "build-args": [ "--share=network" ]
```

Comment:
you can't do this. flathub requires sandboxed builds so this breaks the build: https://flathub.org/builds/#/builders/2/builds/6034